modules/dxp/apps/osb/osb-faro/osb-faro-web/src/main/js/contacts/components/distribution-card/AddPropertyForm.jsx

```diff
@@ -116,7 +117,10 @@ const AddPropertyForm = ({
 
 	return (
 		<Card.Body className='add-property-form-root'>
-			<div className='row d-flex flex-column flex-grow-1'>
+			<div
+				className='row d-flex flex-column flex-grow-1'
+				id={Containers.DistributionBreakdownCard}
+			>
```

Comment:
You should add the id on `<Card>` instead here. Can you update?

Reply:
There's a parent component called `Distribution Card` on `contacts/components/distribution-card/index.tsx`   which is already passing an id in line 185. Should I add it again on this child component?